### PR TITLE
fix(dedicated-ip): arrangments of column and tooltip

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/ip-strat`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #.MANAGER-10289
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.html [hunk 1]
```diff
@@ -185,10 +162,38 @@
...
-                    data-translate="ip_table_header_organisation"
-                ></th>
+                <th scope="col">
+                    <script type="text/ng-template" id="ip_alerts_help.html">
+                        <div
+                            class="text-right"
+                            data-ng-if="$ctrl.hasSpamAlerts"
+                        >
+                            <strong data-trnaslate="ip_alerts_spam_noun"></strong>
+                            <span data-trnaslate="ip_alerts_spam_help"></span>
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/list.html:172]:
```suggestion
                            <strong data-translate="ip_alerts_spam_noun"></strong>
                            <span data-translate="ip_alerts_spam_help"></span>
```

Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/list.html:172]:
```suggestion
                            <strong data-translate="ip_alerts_spam_noun"></strong>
                            <span data-translate="ip_alerts_spam_help"></span>
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.html [hunk 2]
```diff
@@ -185,10 +162,38 @@
...
+                        </div>
+                        <div
+                            class="text-right"
+                            data-ng-if="$ctrl.hasAntihackAlerts"
+                        >
+                            <strong data-trnaslate="ip_alerts_antihack_noun"></strong>
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/list.html:178]:
```suggestion
                            <strong data-translate="ip_alerts_antihack_noun"></strong>
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.html [hunk 3]
```diff
@@ -185,10 +162,38 @@
...
+                            <span data-trnaslate="ip_alerts_antihack_help"></span>
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/list.html:179]:
```suggestion
                            <span data-translate="ip_alerts_antihack_help"></span>
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.html [hunk 4]
```diff
@@ -185,10 +162,38 @@
...
+                        </div>
+                        <div
+                            class="text-right"
+                            data-ng-if="$ctrl.hasMitigationAlerts"
+                        >
+                            <strong data-trnaslate="ip_alerts_mitigation_noun"></strong>
+                            <span data-trnaslate="ip_alerts_mitigation_help"></span>
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/list.html:186]:
```suggestion
                            <strong data-translate="ip_alerts_mitigation_noun"></strong>
                            <span data-translate="ip_alerts_mitigation_help"></span>
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/ip/components/list/list.html (hunk 2 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/ip/components/list/list.html]: Done Anoop

**Review by anooparveti — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

**Pull request merged**